tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,16 @@
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
+  init do
+    log "Make sure acpi-cpufreq is enabled" 
+    run_cmd('modprobe acpi-cpufreq')
+  end
+
+  run do
+    num_procs = run_cmd('cat /proc/cpuinfo | grep processor | wc -l')
+    (0..(num_procs.to_i - 1)).each do |n|
```

Comment:
You may prefer using `Dir.glob('/sys/devices/system/cpu/cpu[0-9]*').each do |dirname|` as a simpler and more readable alternative